### PR TITLE
Order latest translations & drafts by language name

### PR DESCRIPTION
**Pull request by dsgoers**

### CODE REVIEW

**Comment by dsgoers** [comment]
bump 😄

File: app/models/resource.rb [hunk 1]
```diff
@@ -56,6 +56,7 @@ def latest(is_published = [true, false])
                        on translations.version = max_table.max_version
                        and translations.language_id = max_table.language_id
                        and translations.resource_id = max_table.resource_id")
+               .includes(:language).order('languages.name ASC')
```
Comment (twinge) [inline app/models/resource.rb]: 'ASC' is the default, you don't need to specify it.

Comment (dsgoers) [reply app/models/resource.rb]: Yea thanks, I just like to be specific 😄

**Comment by dsgoers** [comment]
thanks!!

**Pull request merged**